### PR TITLE
Add Day 1 Schedule

### PR DESCRIPTION
**Pull request by selynna**

### CODE REVIEW

**Comment by zachlatta** [comment]
:raised_hands:

On Wed, Jun 1, 2016 at 2:01 PM, Selynna Sun notifications@github.com
wrote:

> ---
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/hackclub/hackcamp/pull/647
> Commit Summary
> - Add Day 1 Schedule
> 
> File Changes
> - _A_ 2016/playbook/ACTIVITIES.md
>   https://github.com/hackclub/hackcamp/pull/647/files#diff-0 (2257)
> - _A_ 2016/playbook/SCHEDULE.md
>   https://github.com/hackclub/hackcamp/pull/647/files#diff-1 (110)
> 
> Patch Links:
> - https://github.com/hackclub/hackcamp/pull/647.patch
> - https://github.com/hackclub/hackcamp/pull/647.diff
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/hackclub/hackcamp/pull/647, or mute the thread
> https://github.com/notifications/unsubscribe/AA8j-C9E06tqXx_rg9dXTqf0obpTBPeJks5qHfMbgaJpZM4Ir-Q8
> .

## 

Zach Latta
Hack Club | zach@hackclub.com

File: 2016/playbook/ACTIVITIES.md [hunk 1]
```diff
@@ -0,0 +1,1788 @@
...
+
+Everything at Hack Camp is optional. You do not have to do anything. In Hack
+Camp we will be following a set schedule, and if you'd like to deviate, please
+do so. However, if you don't follow along with the workshops or don't build
+projects on our schedule, you may fall behind - the choice is up to you.
+
+### Why do we adopt this philosophy?
+
+The objective of Hack Camp is to build an set of materials that allows any
+hackEDU team member to reproduce an experience that that brings people from no
```
Comment (paked) [inline 2016/playbook/ACTIVITIES.md]: > hackEDU

File: 2016/playbook/ACTIVITIES.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+|   1  | http://goo.gl/forms/3CGR5eKncZrvPJsi2 |
+
+
+# Using Slack
+
+We've gone ahead and created a Slack for Hack Camp over at
+https://starthackclub.slack.com to use as a place for communicating with hackers
+during the camp.
+
+
```
Comment (maxwofford) [inline 2016/playbook/ACTIVITIES.md]: extra newlines

File: 2016/playbook/ACTIVITIES.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+progresses. 
+
+All of the feedback form responses will be put into the appropriate sheet in
+https://docs.google.com/a/hackclub.com/spreadsheets/d/1CdCZLincwwsUd_DRQmB6gPecUnn3KN235zEag5bNgDA/edit?usp=drive_web.
+
+| Day  | Link                                  |
+| ---- | --------------------------------------|
+|   1  | http://goo.gl/forms/3CGR5eKncZrvPJsi2 |
+
+
```
Comment (maxwofford) [inline 2016/playbook/ACTIVITIES.md]: extra newlines

File: 2016/playbook/ACTIVITIES.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+"egg" stage.  There will be 3 rounds of rock paper scissors total, so
+egg1 - egg2, egg3 - egg4, egg5 - egg6.  There will then be 3 winners, so
+chicken1 egg2 egg3 chicken4 egg5 chicken6, for example.  Then, there will
+be 2 rounds going on, since there are an odd number of chickens and eggs:
+chicken1 - chicken6, egg2 - egg3, while chicken4 and egg5 wait for partners.
+Then when people lose, an example scenario is: egg1, monkey6, chicken2,
+egg3, chicken4, and egg5.  This continues until there are a specific number
+of divine beings you decide upon.
+
+
```
Comment (maxwofford) [inline 2016/playbook/ACTIVITIES.md]: extra newline

File: 2016/playbook/ACTIVITIES.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+    - [Evolution Rock Paper Scissors](#evolution-rock-paper-scissors)
+    - [I Love My Neighbor Who](#i-love-my-neighbor-who)
+    - [Where The West Wind Blows](#where-the-west-wind-blows)
+- [Introduction talk](#introduction-talk)
+- [Workshops](#workshops)
+- [Demos](#demos)
+- [Feedback Forms](#feedback-forms)
+- [Using Slack](#using-slack)
+
+
```
Comment (maxwofford) [inline 2016/playbook/ACTIVITIES.md]: extra newline

**Comment by selynna** [comment]
@MaxWofford addressed comments and removed newlines

**Comment by maxwofford** [comment]
Fantastic!

**Comment by maxwofford** [comment]
Just moved this over to #651

**Pull request closed without merging**